From a56cbd7a4a4ded63ac7a22e2813e710ecc5f4de7 Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Mon, 19 Jan 2015 17:15:05 +0100
Subject: [PATCH 085/102] sparsify: remove checks for ':' in filenames

Now libguestfs and virt-sparsify itself handle them correctly, so they
can be accepted.
---
 sparsify/sparsify.ml | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/sparsify/sparsify.ml b/sparsify/sparsify.ml
index 84d5a5f..2475bb0 100644
--- a/sparsify/sparsify.ml
+++ b/sparsify/sparsify.ml
@@ -161,16 +161,6 @@ read the man page virt-sparsify(1).
     else
       Sys.getcwd () // indisk in
 
-  let contains_colon filename =
-    try ignore (String.index filename ':'); true with Not_found -> false in
-
-  (* Check filenames don't contain a colon (limitation of qemu-img). *)
-  if contains_colon indisk then
-    error (f_"input filename '%s' contains a colon (':'); qemu-img command line syntax prevents us from using such an image") indisk;
-
-  if contains_colon outdisk then
-    error (f_"output filename '%s' contains a colon (':'); qemu-img command line syntax prevents us from using such an image") outdisk;
-
   (* Check the output is not a char special (RHBZ#1056290). *)
   if is_char_device outdisk then
     error (f_"output '%s' cannot be a character device, it must be a regular file")
-- 
1.9.3

